data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -375,12 +375,16 @@ rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
 
 echo "Starting cluster-bootstrap..."
 
-bootkube_podman_run \
-	--rm \
-	--volume "$PWD:/assets:z" \
-	--volume /etc/kubernetes:/etc/kubernetes:z \
-	"${CLUSTER_BOOTSTRAP_IMAGE}" \
-	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
+if [ ! -f cvo-bootstrap.done ]
+then
+    bootkube_podman_run \
+        --rm \
+        --volume "$PWD:/assets:z" \
+        --volume /etc/kubernetes:/etc/kubernetes:z \
+        "${CLUSTER_BOOTSTRAP_IMAGE}" \
+        start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
+    touch cvo-bootstrap.done
```

Comment:
https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L74

This causes CVO to never run can we change name to `cb-bootstrap.done` or similar?